### PR TITLE
A bit more capabilities and window handles

### PR DESCRIPTION
**Pull request by cryogenian**
+ Made `Capabilities` a `Monoid` and changed type of `Build` monad
+ `withCapabilities` doesn't erase previously added now 
+ replaced `forBrowser` with `version` and `platform`. Reworked `browser`
+ added bindings for `WindowHandle`
+ added firefox profile builder
+ updated docs

@beckyconning Please, review/merge

### CODE REVIEW

File: docs/Selenium.md [hunk 1]
```diff
@@ -203,9 +203,12 @@ getCssValue :: forall e. Element -> String -> Aff (selenium :: SELENIUM | e) Str
 #### `getAttribute`
 
 ``` purescript
-getAttribute :: forall e. Element -> String -> Aff (selenium :: SELENIUM | e) String
+getAttribute :: forall e. Element -> String -> Aff (selenium :: SELENIUM | e) (Maybe String)
 ```
 
+Tries to find an element starting from `document` will return `Nothing` if there
```
Comment (jonsterling) [inline docs/Selenium.md]: Grammar fix: :smile: 

```markdown
Tries to find an element, starting from `document`; will return `Nothing` if there 
is no element that can be found by the locator.
```

File: docs/Selenium/Combinators [hunk 2]
```diff
@@ -0,0 +1,84 @@
+## Module Selenium.Combinators
```
Comment (jonsterling) [inline docs/Selenium/Combinators]: Can this file (and `FPProfile`) be renamed to end in the `.md` file extension?

File: docs/Selenium.md (hunk 1 above)
Comment (cryogenian) [inline docs/Selenium.md]: fixed

File: docs/Selenium/Combinators (hunk 2 above)
Comment (cryogenian) [inline docs/Selenium/Combinators]: fixed

**Comment by jonsterling** [comment]
thanks @cryogenian! :+1: from me. I'll merge it in & release once the travis build finishes.

**Pull request merged**